packages/compass-sidebar/package.json

```diff
@@ -66,7 +66,6 @@
     "@mongodb-js/connection-form": "^1.23.1",
     "@mongodb-js/connection-info": "^0.1.5",
     "@mongodb-js/connection-storage": "^0.8.1",
-    "@mongodb-js/compass-connections": "^1.25.1",
```

Comment:
Is this expected that this disappeared from the deps here?

Reply:
I skipped a beat there (cuz CI just turned green) but yea after looking more closely I noticed that this package got duplicated in both peerDep and dep, must have been from one of the merge conflict.